ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerComponent.m

```diff
@@ -117,8 +117,10 @@ - (void)requestAd {
   [self addSubview:_banner];
   _banner.adUnitID = _unitId;
   [self setRequested:YES];
+
+  RNGoogleMobileAdsBannerComponent * __weak weakSelf = self;
   _banner.paidEventHandler = ^(GADAdValue *_Nonnull value) {
-    [self sendEvent:@"onPaid"
+    [weakSelf sendEvent:@"onPaid"
```

Comment:
The weak reference should be unwrapped. To avoid crashes if self could have been deallocated by the time the block executes.

Reply:
```
  __weak typeof(self) weakSelf = self;
  _banner.paidEventHandler = ^(GADAdValue *_Nonnull value) {
    typeof(self) strongSelf = weakSelf;
    if (strongSelf) {
      [strongSelf sendEvent:@"onPaid"
                    payload:@{
                      @"value" : value.value,
                      @"precision" : @(value.precision),
                      @"currency" : value.currencyCode,
                    }];
    }
  };
```